extensions/color-casket/src/utilities.tsx

```diff
@@ -0,0 +1,41 @@
+import { useEffect } from "react";
```

Comment:
Seems it's better to use `.ts` extension for this file due to lack of JSX in order to avoid extra load on webpack loader

Reply:
@musienkoyuriy We don't use webpack. We use esbuild behind the scenes and the entire extension gets bundled into a single file per command. I'd be surprised if that makes a big difference in compile times...